studio/src/components/federatedgraphs-cards.tsx

```diff
@@ -463,25 +463,39 @@ export const Empty = ({
   let labels = "team=A";
   return (
     <EmptyState
+      className="h-auto"
       icon={<CommandLineIcon />}
-      title="Create federated graph using CLI"
+      title="No graphs found"
       description={
         <>
-          No federated graphs found. Use the CLI tool to create one.{" "}
+          Use the CLI tool to create either a federated graph ({" "}
           <a
             target="_blank"
             rel="noreferrer"
             href={docsBaseURL + "/cli/federated-graphs/create"}
             className="text-primary"
           >
-            Learn more.
-          </a>
+            docs
+          </a>{" "}
+          ) or a monograph ({" "}
+          <a
+            target="_blank"
+            rel="noreferrer"
+            href={docsBaseURL + "/cli/monograph/create"}
+            className="text-primary"
+          >
+            docs
+          </a>{" "}
+          ).
         </>
       }
       actions={
         <div className="flex flex-col gap-y-6">
           <CLI
-            command={`npx wgc federated-graph create production --namespace ${router.query.namespace} --label-matcher ${labels} --routing-url http://localhost:4000/graphql`}
+            command={`npx wgc federated-graph create production --namespace ${router.query.namespace} --label-matcher ${labels} --routing-url http://localhost:3002/graphql`}
```

Comment:
We should show a dropdown from where the user can select the command for the graph type. Showing two long cli commands can be overwhelming.

Reply:
Done